src/jlapi.c

```diff
@@ -5,12 +5,15 @@
   miscellaneous functions for users of libjulia.so, to handle initialization
   and the style of use where julia is not in control most of the time.
 */
+#define __STDC_LIMIT_MACROS
+#define __STDC_CONSTANT_MACROS
 #include "platform.h"
 
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <assert.h>
+#include <llvm-c/Disassembler.h>
```

Comment:
llvm content is not allowed in this file (breaks `make JULIACODEGEN=none` build`